src/content/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration.mdx

```diff
@@ -48,7 +48,7 @@ Follow these troubleshooting tips for Docker or Kubernetes as applicable:
 
     4. Run this NRQL query:
 
-       ```
+       ```sql
        docker logs nri-prometheus | grep "error emitting metrics"
```

Comment:
is this a NRQL query or a docker command?

Reply:
Good catch!